content/tooling/api/scripts.md

```diff
@@ -52,24 +51,98 @@ curl -X PUT "https://api.cloudflare.com/client/v4/accounts/9a7806061c88ada191ed0
 }
 ```
 
```

Comment:
skipping stuff below this because i think it is included in #572 - worried that if any commits happen there that cause this to be different, it's going to be a git headache. luckily i know @ashleymichal is a git master 👍

Reply:
lol! i should probably have marked this one WIP.